content/blog/2021-11-05.md

```diff
@@ -0,0 +1,206 @@
+---
+title: The XMPP Newsletter October 2021
+author: emus
+date: 2021-11-05
+categories: ['Newsletter']
+---
+
+Welcome to the XMPP Newsletter covering the month of October 2021.
+
+Many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, especially throughout the current situation, please consider to say thanks or help these projects!
+
+[Read this Newsletter via our RSS Feed](https://xmpp.org/feeds/all.atom.xml)!
+
+Interested in supporting the Newsletter team? Read more at the bottom.
+
+Other than that - enjoy reading!
```

Comment:
Might change this in whatever template you use to an em-dash. Here it is for copy pasting: "—".